api/nakadi-event-bus-api.yaml

```diff
@@ -2215,6 +2269,8 @@ definitions:
           contains at least one of the scopes in this list.
           If no scopes provided then anyone can publish to this event type.
 
+          Usage of write_scopes is deprecated.
```

Comment:
It will be ignored, and hopefully eventually removed. In Zalando deployments, the feature has been disabled months ago, so they are already ignored.